docs/docs/guides/labs/components/using-environment-variables-in-components.md

```diff
@@ -0,0 +1,89 @@
+---
+title: 'Using environment variables with components'
+sidebar_position: 100
+---
+
+
+import Preview from '@site/docs/partials/\_Preview.md';
+
+<Preview />
+
+
+It's common to want to configure components depending on the environment in which they are run. `dg` and components make it easy to configure this behavior.
+
+
+## Setting up a components project
+
+To demonstrate how to configure components for different environments, we'll set up a basic ELT pipeline using Sling in an empty Dagster components project:
+
+<CliInvocationExample path="docs_snippets/docs_snippets/guides/dg/using-env/1-dg-init.txt" />
+
+<CliInvocationExample path="docs_snippets/docs_snippets/guides/dg/using-env/2-activate-venv.txt" />
+
+We'll install `dagster-sling` and scaffold an empty Sling connection component:
+
+<CliInvocationExample path="docs_snippets/docs_snippets/guides/dg/using-env/3-uv-add-sling.txt" />
+
+<CliInvocationExample path="docs_snippets/docs_snippets/guides/dg/using-env/4-dg-list-component-types.txt" />
```

Comment:
not sure if this is blocking or not but this table looks kind of weirdly cut off on my macbook 

![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/KkgkhQiz3H2YHdisAdzD/13bc439d-a927-4b33-b3fe-a555efc5b6ef.png)